plugin.audio.hearthis_at/addon.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.hearthis_at" name="Hearthis.at" version="0.1" provider-name="Eike Baran">
```

Comment:
This is important has well, please rename your addon version so that it is in the format X.X.X (0.1.0 for instance).  Have a read here for the version conventions:
http://kodi.wiki/view/Addon.xml#version_attribute